tests/unit/query_plan_match_filter_return.cpp

```diff
@@ -406,6 +410,101 @@ TEST_F(ExpandFixture, Expand) {
   EXPECT_EQ(4, test_expand(EdgeAtom::Direction::IN, memgraph::storage::View::OLD));
   EXPECT_EQ(8, test_expand(EdgeAtom::Direction::BOTH, memgraph::storage::View::OLD));
 }
+class TestAuthChecker : public memgraph::query::AuthChecker {
+ public:
+  MOCK_CONST_METHOD2(IsUserAuthorized, bool(const std::optional<std::string> &username,
+                                            const std::vector<memgraph::query::AuthQuery::Privilege> &privileges));
+
+  bool Accept(const memgraph::auth::User &user, const memgraph::query::DbAccessor &dba,
+              const memgraph::query::VertexAccessor &vertex, const memgraph::storage::View &view) const final {
+    auto maybe_labels = vertex.Labels(view);
+    if (maybe_labels.HasError()) {
+      switch (maybe_labels.GetError()) {
+        case memgraph::storage::Error::DELETED_OBJECT:
+          throw memgraph::query::QueryRuntimeException("Trying to get labels from a deleted node.");
+        case memgraph::storage::Error::NONEXISTENT_OBJECT:
+          throw memgraph::query::QueryRuntimeException("Trying to get labels from a node that doesn't exist.");
+        case memgraph::storage::Error::SERIALIZATION_ERROR:
+        case memgraph::storage::Error::VERTEX_HAS_EDGES:
+        case memgraph::storage::Error::PROPERTIES_DISABLED:
+          throw memgraph::query::QueryRuntimeException("Unexpected error when getting labels.");
+      }
+    }
+
```

Comment:
This is the copied class, right? (no action item, just for my information)

Reply:
Whole if is copied.